core/assets/jsons/save.json

```diff
@@ -49,80 +49,129 @@
     "path": "levels/shooter-tutorial.json"
   },
   {
-    "name": "level 5",
+    "name": "Cave level 8",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
     "path": "levels/easyLevel.json"
   },
   {
-    "name": "level 6",
+    "name": "Cave level 9",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
     "path": "levels/mediumLevel.json"
   },
   {
-    "name": "level 7",
+    "name": "Cave level 10",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
     "path": "levels/hardLevel.json"
   },
   {
-    "name": "level 8",
+    "name": "Cave level 11",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
-    "path": "levels/enemySurroundingPlayer.json"
+    "path": "levels/enemySurroundingPlayerUpdated.json"
   },
   {
-    "name": "level 9",
+    "name": "Cave level 12",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
-    "path": "levels/forceSneak.json"
+    "path": "levels/forceSneakUpdated.json"
   },
   {
-    "name": "level 10",
+    "name": "Cave level 13",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
-    "path": "levels/shooterAndPatrol.json"
+    "path": "levels/Jalil-11.json"
   },
   {
-    "name":       "level 11",
+    "name":       "Cave level 14",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
-    "path":       "levels/maxLevel.json"
+    "path":       "levels/medium.json"
   },
   {
-    "name": "level 12",
+    "name": "Cave level 15",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
-    "path": "levels/onlypatrol.json"
+    "path": "levels/Jalil-12.json"
   },
   {
-    "name": "level 13",
+    "name": "Cave level 16",
+    "world":      0,
+    "unlocked":   true,
+    "completed":  true,
+    "path": "levels/Jalil-14.json"
+  },
+  {
+    "name": "Fores level 1",
+    "world":      1,
+    "unlocked":   true,
+    "completed":  true,
+    "path": "levels/bryansfirstlevel.json"
+  },
+  {
+    "name": "Forest level 2",
+    "world":      1,
+    "unlocked":   true,
+    "completed":  true,
+    "path": "levels/onlypatrolUpdated.json"
+  },
+  {
+    "name": "Forest level 3",
     "world":      1,
     "unlocked":   true,
     "completed":  true,
-    "path": "levels/ronnie-level.json"
+    "path": "levels/lauraforest.json"
   },
   {
-    "name": "level 14",
+    "name": "Forest level 4",
+    "world":      1,
+    "unlocked":   true,
+    "completed":  true,
+    "path": "levels/Jalil-15.json"
```

Comment:
Is there supposed to be gaps in the trees for the player to walk through? Because rn there are

Reply:
idk which level you mean, this is a cave one